README.md

```diff
@@ -1,6 +1,6 @@
 # XLIFF Sync
 
-[![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](https://opensource.org/licenses/MIT) [![Version](https://vsmarketplacebadges.dev/version-short/rvanbekkum.xliff-sync.svg?color=blue)](https://marketplace.visualstudio.com/items?itemName=rvanbekkum.xliff-sync) [![Installs](https://vsmarketplacebadges.dev/installs/rvanbekkum.xliff-sync.svg?color=blue)](https://marketplace.visualstudio.com/items?itemName=rvanbekkum.xliff-sync)
+[![License: MIT](https://img.shields.io/badge/License-MIT-blue.png)](https://opensource.org/licenses/MIT)
```

Comment:
Any particular reason for changing this in the README?

Reply:
i.e. changing it from svg to png and removing the other properties?

Reply:
ah yes it wouldn't let me create the vsix package otherwise (doesnt support svg in readme) at least tool I used to build (vsce). But as this wasn't a problem for you I just revert the changes.